libs/langchain-community/langchain.config.js

```diff
@@ -134,6 +134,7 @@ export const config = {
     "vectorstores/hanavector": "vectorstores/hanavector",
     "vectorstores/lancedb": "vectorstores/lancedb",
     "vectorstores/libsql": "vectorstores/libsql",
+    "vectorstores/mariadb": "vectorstores/mariadb",
```

Comment:
You will also need to add this to `requiresOptionalDependencies` below

Reply:
right, adding